test_codecept/e2e/config/appTestConfig.js

```diff
@@ -152,6 +152,12 @@ const data = {
                 email: "xui_hearing_manager_cr84_off@justice.gov.uk",
                 release: "hearing_CR84", userIdentifier: "HEARING_MANAGER_CR84_OFF",
                 key: "Monday01"
+            },
+            {
+                idamId: "11111111-1111-1111-1111-111111111111",
```

Comment:
We really need to get these into the key vault. This stuff is all going into a public github repo remember. It's a miracle that some security person hasn't come down on us for this